src/signal_generator.cpp

```diff
@@ -1412,19 +1412,18 @@ void SignalGenerator::stop()
 
 void SignalGenerator::startStop(bool pressed)
 {
+	if (amp1 && amp2) {
```

Comment:
When a channel is enabled or disabled from the GUI, while the Signal Generator is running, this function does not get called again. 
If a channel is enabled/disabled while running, the method **stop**() is called, then **start**() to restart the acquisition. In this case, should we move the "powerdown" controls in the start() and stop() function?